app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -3980,4 +3980,78 @@ class PatientRepositoryAndroidTest {
         patientWithAnotherFacilityAsAssignedFacility, patientWithCurrentFacilityAsAssignedFacility
     )
   }
+
+  @Test
+  fun querying_all_patients_in_facility_must_return_list_of_patients_partitioned_by_assigned_facility_and_order_by_name() {
+    fun createPatientWithNameAndAssignedFacilityID(
+        patientUuid: UUID,
+        patientName: String,
+        assignedFacilityId: UUID?
+    ) {
+      val patientProfile = testData
+          .patientProfile(patientUuid = patientUuid, patientName = patientName, patientAssignedFacilityId = assignedFacilityId)
+          .let { patientProfile ->
+            patientProfile.copy(
+                patient = patientProfile.patient.copy(fullName = patientName, assignedFacilityId = assignedFacilityId),
+            )
+          }
+
+      patientRepository.save(listOf(patientProfile)).blockingAwait()
+    }
+
+    fun searchResults(facility: Facility): List<String> {
+      return patientRepository
+          .allPatientsInFacility2(facility)
+          .map { it.fullName }
+    }
+
+    // given
+    val currentFacility = TestData.facility(
+        uuid = UUID.fromString("c0056c11-105d-4079-80fa-c745128f7fc5"),
+        name = "CHC Bucho"
+    )
+    val facility2 = TestData.facility(
```

Comment:
Since we have one facility named `currentFacility`, can we name the other to `otherFacility`? This also removes confusion when reading `patientWithCurrentFacilityAsAssignedFacility2` because I initially thought this patient was assigned to `faciltiy2` instead of `patient 2 assigned to current facility`.

Reply:
Good point will do that 👍🏽